src/compiler/crystal/semantic/cleanup_transformer.cr

```diff
@@ -126,6 +126,18 @@ module Crystal
       {@last_is_truthy, @last_is_falsey}
     end
 
+    def transform(node : Alias)
```

Comment:
This is incorrect, as `node` is the alias definition itself. The actual use site is simply a `Path`.

Reply:
You could leave out aliases for now due to the extra complications in #11043.

Reply:
Good call, I'll work on a separate PR for deprecating aliases and types.